src/ui_fsmenu/addons/manager.cc

```diff
@@ -68,13 +68,14 @@ std::string underline_tag(const std::string& text) {
 std::string filesize_string(const uint32_t bytes) {
 	if (bytes > 1000000000) {
```

Comment:
while you are at it: maybe explain these magic numbers by multipliying: 1000 * 1000 * 1000? and shouldnt it be rather 1024 * 1024 * 1024?

Reply:
Changed the constants to `1000*1000*1000` style. 

If we used 1024 here, the resulting units would be Kibibyte, Mibibyte, and Gibibyte (kiB, MiB, GiB) instead of Kilobyte, Megabyte, Gigabyte (kB, MB, GB). These strings are meant for the end-users rather than for technical purposes, and I believe the 10-based units are more familiar to most users than their 2-based counterparts, so IMHO we should stick with 10-based.